db/migrate/20210115203759_create_articles.rb

```diff
@@ -3,6 +3,7 @@ def change
     create_table :articles do |t|
       t.string :title
       t.text :body
+      t.references :user, null: false, foreign_key: true
```

Comment:
@idprietop is this a good approach? what if I ran this migration already?

Reply:
It is not a good approach, ideally the migration should not be edited later to maintain data integrity.

Reply:
@idprietop that's correct! did you fix this?